lib/rucio/db/sqla/migrate_repo/versions/01eaf73ab656_add_new_rule_notification_state_progress.py

```diff
@@ -39,7 +39,7 @@ def upgrade():
 
     elif context.get_context().dialect.name == 'postgresql':
         schema = context.get_context().version_table_schema + '.' if context.get_context().version_table_schema else ''
-        op.execute('ALTER TABLE ' + schema + 'rules ALTER COLUMN notification TYPE CHAR')
+        op.execute('ALTER TABLE ' + schema + 'rules ALTER COLUMN notification TYPE CHAR')  # pylint: disable=no-member
```

Comment:
To avoid writing this exclusion in the future,  the class `alembic.op` can be added in `pylintrc`:`ignored-classes`.

Reply:
Thx!